persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -577,3 +580,13 @@ markMessagesAsDecryptionResolved:
 UPDATE MessageFailedToDecryptContent
 SET is_decryption_resolved = 1
 WHERE message_id IN ?;
+
+updateMessagesTableAfterOneIsSent {
```

Comment:
```suggestion
updateMessageStatus {
```

I think we can improve the naming. I think the important that this query does is to mark the message as sent.